network/bridge_endpointclient_linux.go

```diff
@@ -126,7 +124,6 @@ func (client *LinuxBridgeEndpointClient) DeleteEndpointRules(ep *endpoint) {
 	for _, ipAddr := range ep.IPAddresses {
 		if ipAddr.IP.To4() != nil {
 			// Delete ARP reply rule.
-			logger.Info("Deleting ARP reply rule for IP address on", zap.String("address", ipAddr.String()), zap.String("id", ep.Id))
```

Comment:
Shouldn't we delete the same lines from `AddEndpointRules()`?

Reply:
@QxBytes lets remove the bridge mode from cni codebase.. its not used today